lib/pages/chat/chat_event_list.dart

```diff
@@ -102,7 +102,10 @@ class ChatEventList extends StatelessWidget {
 
           // The message at this index:
           final event = controller.timeline!.events[i - 1];
-
+          final previousEvent = i > 2 ? controller.timeline!.events[i - 2] : null;
```

Comment:
what is `2`? for the same convention in the future, dont use magic number, please use constant to make the code readable

Reply:
events are displayed in reverse and current event is index - 1, previous is index - 2